troubleshooting/application-title-in-email-templates-wrong.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Application.Title in Email Template is Wrong
+description: You renamed the name of your project in Telerik Platform portal, but in emails Application.Title still the old name appears.
+type: troubleshooting
+page_title: How to Change the Application.Title in Email Templates
+slug: application-title-in-email-templates-wrong
+position:
+tags: backendservices, portal, email-notification
+teampulseid:
+ticketid: 1111828, 1072170, 1060801, 1054819, 1087785, 1100004, 1058035, 1087785
+pitsid:
+
+---
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>{{site.tp}}, {{site.bs}} {{site.bs-n}}</td>
+  </tr>
+</table>
+
+## Description
+After changing the application name in Telerik Platform portal, the old name {{Application.Title}} still appearing in all automated emails.

```

Comment:
To visualize {{ and }} you need to use the unicode symbol, otherwise it tries to replace the content with a variable.